src/libraries/System.Linq.Expressions/src/System/Linq/Expressions/Compiler/AssemblyGen.cs

```diff
@@ -58,6 +58,8 @@ private TypeBuilder DefineType(string name, [DynamicallyAccessedMembers(Dynamica
             return _myModule.DefineType(name, attr, parent);
         }
 
+        [UnconditionalSuppressMessage("ReflectionAnalysis", "IL2026:RequiresUnreferencedCode",
+            Justification = "MulticastDelegate has a ctor with RequiresUnreferencedCode, but the generated derived type doesn't reference this ctor, so this is trim compatible.")]
```

Comment:
@MichalStrehovsky - can you please look at this one? Delegates are magic to me still.